lib/precompiled_assets/manifest.rb

```diff
@@ -29,6 +29,11 @@ def expired?
       mtime && mtime != fetch_mtime
     end
 
+    def inspect
+      truncated_parsed_manifest = "#{@parsed_manifest.to_s[0..7]}..."
+      "<#{self.class.name}:#{object_id} @pathname=#{@pathname.inspect} @parsed_manifest=#{truncated_parsed_manifest}>"
```

Comment:
Thanks for your suggestion. A useful inspection does make sense.
However, I think there is room for improvement here.

* Convention for object inspection is to start with `#<`, not `<#`.
* If the manifest has not yet been parsed this inspects with `@parsed_manifest=...` instead of `@pared_manifest=...`.
* I'm not sure how helpful the first 8 characters of the manifest.json can be. Maybe we should show more characters, or offer some kind of aggregation, like `@parsed_manifest=(23 entries)`.
* The `object_id` is not actually put directly into inspection by default, but implementation varies across Ruby versions, so maybe don't want to go there. I'm fine with keeping it as implemented, or not displaying it at all.

```suggestion
      manifest_inspection = @parsed_manifest.inspect.sub(/\A(.{40}).+\z/, '\1...')
      "#<{self.class.name}:#{object_id} @pathname=#{@pathname.inspect} @parsed_manifest=#{manifest_inspection}>"
```

Reply:
* Good catch
* :+1: 
* I like the `@parsed_manifest=(23 entries)` suggestion :+1:
* I've been here https://bugs.ruby-lang.org/issues/17199 before, and I just used object_id :D. A little bit weird for Ruby to not have a straightforward way to override #inspect.

Since I'm running a patched version anyways, I'll look into adapting this PR with your suggestions.